package.json

```diff
@@ -11,10 +11,10 @@
     "fastify-error": "^0.2.0",
     "fastify-plugin": "^3.0.0",
     "hexoid": "^1.0.0",
-    "stream-wormhole": "^1.1.0"
+    "stream-wormhole": "^1.1.0",
+    "@types/busboy": "^0.2.3"
```

Comment:
I would prefer not shipping anything from definitelyTyped as they are often incomplete and out of the control of the maintainer. I would just let folks know to install them as devDeps when using ts.

Reply:
As we have in other plugins ( as well as on fastify core ) we do not add typescript types to the dependency list. The recommended solution is to add these types to the readme and instruct users to install them (sorta like a peer dep but we also don't use that npm feature because of its inconsistent behavior).

I know this isn't the best solution but its what we compromised and we should continue to stick to it until something better comes along in the future 😁

Reply:
https://github.com/fastify/fastify/blob/0501b3a189b309fc6e81b00046f80efdccdfd521/types/logger.d.ts#L2-L11

Then I prefer to duplicate the type definition as in fastify core.